hw3.js

```diff
@@ -0,0 +1,206 @@
+// #### Task 1 💻
+
+// Создайте переменные и присвойте им значения:
+
+const one = 'true';
+const two = false;
+const three = 17;
+const four = undefined;
+const five = null;
+
+// > Затем с помощью оператора **`typeof`** выведите в консоль все типы созданных значений переменных.
+
+console.log(typeof one,typeof two, typeof three, typeof four,typeof five);
+
+// #### Task 2 💻
+
+// Даны две переменные:
+
+let height = 15;
+let width = 20;
+
+// > С помощью условий выведите в консоль разработчика наибольшее число.
+
+// example one:
+if (height > width) {
+    console.log(height);
+} else console.log(width);
+// example two:
+switch (height) {
+    case height > width: 
+        console.log(height);
+        break;
+
+    default:
+        console.log(width);
+        break;
+};
+// example three:
+console.log(height > width ? height : width);
+
+// #### Task 3 💻
+
+// Написать перебор от 1 до 20, где выведутся все числа кратные трём.
+
+// > Для вычисления кратности обратите внимание на оператор %
+
+for (let i = 1; i <= 20; i++) {
+    if (i % 3 === 0) {
+        console.log(i);
+    }
+};
+
+// #### Task 4 💻
+
+// Нам надо идти на работу, но нужно проверить все ли мы взяли или нет?
+// Нам точно нужны ключи документы и ручка, но из еды нам надо яблоко или апельсин. 
+// Для решения этой задачи нам помогут логические операторы **`|| &&`**.
+
+let key = true;
+let documents = true;
+let pen = true;
+let apple = false;
+let orange = true;
+
+// > Ответ поместите в переменную **shouldGoToWork**
+
+let shouldGoToWork
+
+if (key && documents && pen && (apple || orange)) {
```

Comment:
you can simplify it to shouldGoToWork = (key && documents && pen && (apple || orange)); without if else